src/renderer/components/table/react-table.module.css

```diff
@@ -0,0 +1,43 @@
+.table {
```

Comment:
This file named with `.module.css` ending for some reason?

Reply:
Ok, I've seen some notice in `mocks.d.ts` but.. why the file extension not `.scss`?

Reply:
It is possible to use with `.scss` but this way not working in current PR. I was intentionally went thought css-only way to test styling without use of SCSS and check the possibility to remove it someday. I guess the main benefit of using scss nowadays is nested properties. But now we have [postcss nested plugin](https://github.com/postcss/postcss-nested) which can be used to replace this feature (not implemented also in this PR).

Reply:
Could you remind me why we need `postcss` at all?

Reply:
It is used by `tailwind` for cases when styling goes into separate files `*.css`. We using `@apply` directive in there which is compiled by `postcss` into regular css...